sublime_jedi/completion.py

```diff
@@ -160,15 +197,15 @@ def on_query_completions(self, prefix, locations):
 
         if self._last_location != locations[0]:
             self._last_location = locations[0]
-            ask_daemon(
+            debounced_ask_daemon(
                 self.view,
                 self._receive_completions,
                 'autocomplete',
                 location=locations[0],
             )
             return [], PLUGIN_ONLY_COMPLETION
 
-        if self._last_location == locations[0] and self._completions:
+        if self._last_location == locations[0]:
```

Comment:
I don't remember why it was here. Why we changed that? :)

Reply:
If the completion has nothing inside is not event sent. So this is unnecessary here.